misc/python/materialize/checks/checks.py

```diff
@@ -20,6 +21,7 @@ class Check:
     def __init__(self, base_version: MzVersion, rng: Random | None) -> None:
         self.base_version = base_version
         self.rng = rng
+        self.disabled = False
```

Comment:
Now that I think about it, maybe `self.enabled = True` is better than double negation.

Reply:
Sure, happy to adjust that